app/models/ability.rb

```diff
@@ -70,7 +70,7 @@ def initialize(user)
       (user_is_admin_of?(group.id) && group.enabled_beta_features.include?('export'))
     end
 
-    can [:members_autocomplete, :set_volume, :see_members, :move_discussions_to, :view_previous_proposals], Group do |group|
+    can [:members_autocomplete, :set_volume, :add_discussion, :see_members, :move_discussions_to, :view_previous_proposals], Group do |group|
```

Comment:
these add_ methods are different to :create, discussion because you don't pass the discussion object eh?